frontend/test/metabase/scenarios/question/view.cy.spec.js

```diff
@@ -72,4 +104,117 @@ describe("scenarios > question > view", () => {
       cy.contains("Showing 91 rows");
     });
   });
+
+  describe("apply filters without data permissions", () => {
+    before(() => {
+      // All users upgraded to collection view access
+      signInAsAdmin();
+      cy.visit("/admin/permissions/collections");
+      cy.get(".Icon-close")
+        .first()
+        .click();
+      cy.findByText("View collection").click();
+      cy.findByText("Save Changes").click();
+      cy.findByText("Yes").click();
+
+      // Native query saved in dasbhoard
+      cy.request("POST", "/api/dashboard", {
+        name: "Dashboard",
+      });
+      cy.request("POST", "/api/card", {
+        name: "Question",
+        dataset_query: {
+          type: "native",
+          native: {
+            query: "select * from products where {{category}} and {{vendor}}",
+            "template-tags": {
+              category: {
+                id: "6b8b10ef-0104-1047-1e5v-2492d5954555",
+                name: "category",
+                "display-name": "CATEGORY",
+                type: "dimension",
+                dimension: ["field-id", 4],
```

Comment:
I added a helper to get field/table ids since I _think_ there's some risk in these changing if our sync process updates.  https://github.com/metabase/metabase/blob/master/frontend/test/__support__/cypress.js#L95-L108

Ideally, we'd be able to import the object directly, but the way I wrote it you have to pass a function to receive it:
```js
withSampleDataset(SAMPLE_DATASET => {
  SAMPLE_DATASET.ORDERS.CATEGORY // would be 4  here
})
```

Reply:
I made these changes, but the request is failing; could you look at the "sub in withSampleDataset" PR to see if I did something wrong?

Reply:
Whoops! I think my example code misled you. `category` and `vendor` are on `products` not `orders`. I didn't run the code, but that was the only thing I spotted.

Reply:
Oh, haha, that's obvious. My bad.